### DISCUSSION TITLE
Adding second map region

### DISCUSSION THREAD

**Discussion by vhmoura** [osm-search/Nominatim#2416] [General]
Hi,
I am currently running Nominatim with UK/Ireland maps and normally run updates every couple of days to get the newest added details.
I would like to know how can I add a second map region, for example South Africa and Australia. Also, would the daily updates pick up both maps?

Thanks

**Comment by mtmail** [comment]
If you're using Nominatim 3.6 or earlier the `utils/import_multiple_regions.sh` script can help. Check the corresponding documentation https://www.nominatim.org/release-docs/3.6.0/admin/Advanced-Installations/

But needs changes to work with Nominatim 3.7  https://github.com/osm-search/Nominatim/issues/2365 so currently not usable with the latest version.
